docs/guides/backends.rst

```diff
@@ -0,0 +1,184 @@
+
+.. TODO: updating the link to the backend test excluding RH gates to braket when public
+
+There is also the case where a certain backend doesn't use some gates that we support. When this happens, in the backend test, a list of excluded gates will be used in the ``QuantumSimulatorGatesTest`` like in `TestCirqSimulatorGates <https://github.com/zapatacomputing/orquestra-cirq/blob/main/tests/orquestra/integrations/cirq/simulator/simulator_test.py#L147>`_. For example, that might look something like this:
+
+.. code-block:: python
+
+  class TestCirqSimulatorGates(QuantumSimulatorGatesTest):
+      gates_to_exclude = ["RH"]
+      pass
+
+.. _backend_methods:
+
+Backend Methods Usage
+=====================
+
+In order to use quantum hardware, the user must use the backend's authentication method to prove that they have the right credentials to send instruction to the hardware. The authentication method is specified by the hardware provider. Providing authentication credentials is the first step in using ``QuantumBackend``. Below is an example of using IBM's hardware through ``QiskitBackend``.
+
+``QuantumSimulator``\ s run in your own environment and therefore can be used without needing authentication credentials.
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend creation example
+  :end-before: # End QuantumBackend creation example
+
+General Backend Methods
+-----------------------
+
+After intializing the backend we can send the ``Circuit`` and number of repetitions to the backend and read the measurement. For more information on creating circuits to send to a backend, refer to the :ref:`Circuits Guide <circuits_guide>`. For a single circuit, that is executed as follows:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend run and measure circuit
+  :end-before: # End QuantumBackend run and measure circuit
+
+If you want to run multiple circuits with different number of samples, the code needs to be modified as follows:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend run and measure circuitset
+  :end-before: # End QuantumBackend run and measure circuitset
+
+``QuantumBackend`` can also provide the distribution of the outcome measurement. It can be extracted using the following approach:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # Quantumbackend measurement distribution
+  :end-before: # End Quantumbackend measurement distribution
+
+In all instances, ``QuantumBackend`` automatically adds the measurement operators to the circuit. Therefore, the user is not required to include it in the circuit.
+
+Simulator-specific methods
```

Comment:
I think we're lacking information about using simulations with noise?
It would be good to mention it somewhere, perhaps in this section?

Sth like:
"Some simulators, such as Qiskit or QSim allow for running simulations with various noise models – for more details how to use that, please refer to docs for particular simulator".

Reply:
fixed in 4551c621e5c9c2b2295716c545eb6211b836df64